### PR TITLE
Initial incubator resources for container build

### PR DESCRIPTION
**Pull request by tylerthome**
Completes some items from #627 

### What changes did you make?
  - add Dockerfile definitions for `base` and `app` container images to `.incubator` directory
  - add auxiliary files (e.g. nginx config, entrypoint bash script) to `.incubator` directory
### Rationale behind the changes?
  - prepare for GitHub Action deployment to incubator ECR and ECS
### Testing done for these changes
  - deployment to ECR
### What did you learn or can share that is new?(optional)
  - Dockerized nginx customization for server blocks (without TLS), in contrast to existing EC2 configuration

### Screenshots of Proposed Changes Of The Website  (if any, please do not screen shot code changes)
N/A

### CODE REVIEW

**Review by paulespinosa — APPROVED** [review]
Looks good! Renaming `docker-entrypoint.sh` will help avoid confusion. Thank you!

**Review by paulespinosa — COMMENTED** [review]
`gunicorn` is not a dependency anymore and I can't see where it's being installed. You might be able to get away with running the `fastapi run app/main.py` command instead.

**Comment by tylerthome** [comment]
> `gunicorn` is not a dependency anymore and I can't see where it's being installed. You might be able to get away with running the `fastapi run app/main.py --port 8000` command instead.

Good call @paulespinosa  - I made this change as you suggested. It's using a virtual env there now because there was an issue running either of the `fastapi` and `uvicorn`  commands as such in the entrypoint.sh. They could run with `poetry run` but this caused module references to fail while resolving from import paths. This approach worked as expected in the incubator deployment currently running at https://qa.homeunite.us, but open to any alternative approaches if this looks familiar to anyone.

**Comment by paulespinosa** [comment]
> > `gunicorn` is not a dependency anymore and I can't see where it's being installed. You might be able to get away with running the `fastapi run app/main.py --port 8000` command instead.
> 
> Good call @paulespinosa - I made this change as you suggested. It's using a virtual env there now because there was an issue running either of the `fastapi` and `uvicorn` commands as such in the entrypoint.sh. They could run with `poetry run` but this caused module references to fail while resolving from import paths. This approach worked as expected in the incubator deployment currently running at https://qa.homeunite.us, but open to any alternative approaches if this looks familiar to anyone.

It looks good. Nothing off the top of my head can think of doing it any other way. Nice work!

File: backend/app/health.py [hunk 1]
```diff
@@ -1,8 +1,26 @@
...
 health_router = APIRouter()
 
 
 @health_router.get("/", status_code=status.HTTP_200_OK)
 def health():
     return "UP"
+
+@health_router.get("/nginx-logs", status_code=status.HTTP_200_OK)
+def nginx_logs():
+    with open('/var/log/qa.homeunite.us/nginx-access.log') as f:
```
Comment (paulespinosa) [inline backend/app/health.py:13]: This is a good place to introduce a `HUU_ENVIRONMENT` environment variable in `config.Settings`.

Comment (tylerthome) [reply backend/app/health.py:13]: Ah good catch, will change this before merging

**Pull request merged**